src/test/deploy/functions/runtimes/node/parseTriggers.spec.ts

```diff
@@ -368,6 +368,38 @@ describe("addResourcesToBuild", () => {
     await expect(convertedBackend).to.eventually.deep.equal(expectedBackend);
   });
 
+  it("should expand vpc connector in w/ shorthand form", async () => {
+    const trigger: parseTriggers.TriggerAnnotation = {
+      ...BASIC_TRIGGER,
+      httpsTrigger: {},
+      vpcConnector: "hello-vpc",
+    };
+
+    const result = build.empty();
+    parseTriggers.addResourcesToBuild("project", "nodejs16", trigger, result);
+
+    const expected: build.Build = build.of({
+      func: {
+        ...BASIC_ENDPOINT,
+        httpsTrigger: {},
+        vpc: {
+          connector: "hello-vpc",
+        },
+      },
+    });
+    expect(result).to.deep.equal(expected);
+
+    const expectedBackend: backend.Backend = backend.of({
+      ...BASIC_BACKEND_ENDPOINT,
+      httpsTrigger: {},
+      vpc: {
+        connector: `projects/project/locations/${api.functionsDefaultRegion}/connectors/hello-vpc`,
+      },
+    });
+    const convertedBackend = resolveBackend(expected);
+    await expect(convertedBackend).to.eventually.deep.equal(expectedBackend);
```

Comment:
any reason that you are using `eventually` here instead of awaiting `resolveBackend`?

Reply:
Copy-and-paste! I think I'd done what you said, but will keep this style since other test cases are also written in this style.